config/README.md

```diff
@@ -10,10 +10,13 @@ To reduce efforts and duplication we decided to adapt the generation approach to
 
 ## Overview
 
-The `tools/peribolosbuilder.go` contains our custom generation logic for the peribolos configuration.
+We are using our [community-tools](https://github.com/open-feature/community/) to merge our configuration.
```

Comment:
Should you link to the community or the community tools repo?

Reply:
```suggestion
We are using our [community-tools](https://github.com/open-feature/community-tooling/) to merge our configuration.
```